lib/Fhp/Action/GetSEPADirectDebitParameters.php

```diff
@@ -40,15 +42,25 @@ public static function create(string $seqType, bool $singleDirectDebit, string $
         $result->coreType = $coreType;
         $result->seqType = $seqType;
         $result->singleDirectDebit = $singleDirectDebit;
-
         return $result;
     }
 
     /** {@inheritdoc} */
     public function createRequest(BPD $bpd, ?UPD $upd)
     {
-        $type = $this->singleDirectDebit ? 'HIDSES' : 'HIDMES';
-
+		switch ($this->coreType){
```

Comment:
If you expose `static function getHixxesSegmentName(string $coreType, bool $singleDirectDebit): string` here, you can reuse it in `SendSEPADirectDebit::createRequest()` (and you don't need the `break`s anymore because it's just `return` then).

Reply:
sorry, not clear to me.

Reply:
After you push a commit with the formatting fixed, I can send a commit with what I meant.

Reply:
ok, understand now